liri.js

```diff
@@ -0,0 +1,155 @@
+      log("Tweets failed to load, this is the error: " + error);
+    }
+  });
+}
+
+
+// spotify-this-song
+function spotify() {
+  let spot = new Spotify({
+    id: keys.spotifyKeys.client_ID,
+    secret: keys.spotifyKeys.client_Secret
+  });
+  if (!query) {
+    query = "The Sign"
+  }
+  let params = {
+    type: 'track',
+    query: query
+  };
+  spot.search(params, function(error, response) {
+    if (error) {
+      return console.log('Error occurred: ' + error);
+    } else {
+      // console.log(response.tracks.items[0]);
+      console.log("Artist: " + response.tracks.items[0].album.artists[0].name);
+      log("Artist: " + response.tracks.items[0].album.artists[0].name);
+      console.log("Song: " + response.tracks.items[0].name);
+      log("Song: " + response.tracks.items[0].name);
+      console.log("Preview Link: " + response.tracks.items[0].preview_url);
+      log("Preview Link: " + response.tracks.items[0].preview_url);
+      console.log("Album: " + response.tracks.items[0].album.name);
+      log("Album: " + response.tracks.items[0].album.name);
+    }
+  });
+}
+
+
+// movie-this
+function imdb() {
+  if (!query) {
+    query = 'Mr. Nobody.'
+  }
+  var queryURL = "https://www.omdbapi.com/?t=" + query + "&y=&plot=short&apikey=40e9cece";
+  request.get(queryURL, function(error, response, body) {
+    if (!error && response.statusCode === 200) {
+      // console.log(JSON.parse(body))
+      console.log("Movie Title: " + JSON.parse(body).Title);
+      log("Movie Title: " + JSON.parse(body).Title);
+      console.log("Year Released: " + JSON.parse(body).Year);
+      log("Year Released: " + JSON.parse(body).Year);
+      console.log("IMDB Rating: " + JSON.parse(body).imdbRating);
+      log("IMDB Rating: " + JSON.parse(body).imdbRating);
+      console.log("Rotten Tomatoes Score: " + JSON.parse(body).Ratings[1].Value);
+      log("Rotten Tomatoes Score: " + JSON.parse(body).Ratings[1].Value);
+      console.log("Country: " + JSON.parse(body).Country);
+      log("Country: " + JSON.parse(body).Country);
+      console.log("Language: " + JSON.parse(body).Language);
+      log("Language: " + JSON.parse(body).Language);
+      console.log("Plot: " + JSON.parse(body).Plot);
+      log("Plot: " + JSON.parse(body).Plot);
+      console.log("Actors: " + JSON.parse(body).Actors);
+      log("Actors: " + JSON.parse(body).Actors);
+    } else {
+      console.log("Movie details failed to load, this is the error: " + error);
+      log("----ERROR----Movie details failed to load, this is the error: " + error + "----END ERROR----");
+    }
+  })
+}
+
+
+//do-what-it-says
+function simon() {
+  fs.readFile("random.txt", "utf8", function(error, data) {
+    if (!error) {
+      //console.log(data)
+      var arr = data.split(",");
+      command = arr[0];
+      query = arr[1].replace(/\n$/, "");
+      // console.log(arr)
+      // console.log(query)
+      // console.log(command)
+      main();
```

Comment:
main should take an argument and not refer to a global variable. Now that we are doing server side code we don't need to have global variables at all unless they are a state object.